src/Medulla/Kubernetes/Operator/Operator.csproj.DotSettings

```diff
@@ -0,0 +1,2 @@
+<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
+	<s:String x:Key="/Default/CodeInspection/CSharpLanguageProject/LanguageLevel/@EntryValue">CSharp100</s:String></wpf:ResourceDictionary>
```

Comment:
Not sure why we have a WPF resource dictionary here, is this actually needed? If so why? If not please remove.

Reply:
@keithlogan94  was this added for the database service? I can remove it from this PR.